setup.py

```diff
@@ -2,6 +2,10 @@
 import sys
 from setuptools import setup
 from warnings import warn
+import versioneer
+
+version = versioneer.get_version()
+cmdclass = versioneer.get_cmdclass()
```

Comment:
Similar to `version`, you're not passing `cmdclass` to `setup()` either. Here is the [line](https://github.com/pandas-dev/pandas/blob/master/setup.py#L651) from pandas.